File: src/Auctor/Auctor/Domain/Consultation.cs
namespace VolskNet.Auctor
{
    using System;

    public class Consultation
    {
        public Guid Id { get; set; }
        public string StartTime { get; set; }
        public string EndTime { get; set; }
        public Guid RoomId { get; set; }
        public Guid LecturerId { get; set; }
    }
}

File: src/Auctor/Auctor.App/src/models/consultation.ts
import { CommonUtils } from '../common-utils';

export class Consultation {
  constructor(){
    this.date = new Date();
    this.startTime = `${CommonUtils.getHoursString(this.date)}:${CommonUtils.getMinutesString(this.date)}`;
    this.endTime = `${CommonUtils.getHoursString(this.date)}:${CommonUtils.getMinutesString(this.date)}`;
    this.capacity = 1;
  }

  public id: string;
  public date: Date;
  public startTime: string;
  public endTime: string;
  public roomId: string;
  public lecturerId: string;
  public capacity: number;
}